README.adoc

```diff
@@ -22,14 +22,14 @@ To get started, it is best to have the latest JDK and Gradle installed. The HEAD
 
 #### Building from the command line
 
-Official builds are currently using JDK 11. Our GitHub Actions build and test with JDK 11 and JDK 17.
```

Comment:
> Official builds are currently using JDK 11

I'd rather defer this particular change to a later PR. It's not part of "require JDK 17+ for wallettemplate".

Reply:
Fixed.